validate-config/validate-config.yaml

```diff
@@ -0,0 +1,65 @@
+name: Hub Config Validation (R)
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: main
+    paths:
+      - 'hub-config/**'
+      - '!**README**'
+
+jobs:
+  validate-submission:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          install-r: false
+          use-public-rspm: true
+
+      - name: Update R
+        run: |
+          sudo apt-get update
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          packages: any::gt, hubverse-org/hubValidations, any::sessioninfo
```

Comment:
> Until they are on CRAN but given we now have an r-universe, should we be using that to install from instead of GitHub for the time being?

Oh we can actually set the R-universe repo in the `setup-r` step!
https://github.com/r-lib/actions/tree/v2-branch/setup-r-dependencies#dependencies-in-other-cran-like-repositories

Reply:
Given that this will affect all of the workflows, I am going to create a separate PR for it after this goes through.